src/Gir.Tests/SymbolTableTests.cs

```diff
@@ -48,12 +48,13 @@ public void NoAliasTypeAfterProcessing ()
 		}
 
 		[Test]
+		[Ignore("there is no void*")]
```

Comment:
This test can probably be removed as there are no pointers inside Type.Name

Reply:
Lets remove it. `Ignore`d tests seem to cause problems with CI for right now.